app/views/posts/edit.html.haml

```diff
@@ -0,0 +1,5 @@
+=simple_form_for @post do |f|
+	=f.input :title
```

Comment:
Nitpicky here, but can we use spaces instead of tabs?  Otherwise we'd be inconsistent across the files and it would be a challenge.